tests/testthat/test-helpers.R

```diff
@@ -17,3 +17,8 @@ test_that('tictoc', {
 test_that('log with separator', {
     expect_output(log_with_separator(42), '===')
 })
+
+test_that('log failure', {
+  expect_output(log_failure("foobar"), NA)
+  expect_output(log_failure(foobar), 'ERROR.*foobar')
+})
```

Comment:
```suggestion
  expect_error(log_failure('foobar'), NA)
  expect_error(log_failure(foobar))
})
```

Reply:
a bit confused since both 
`expect_error(log_failure("foobar"), NA)`
`expect_output(log_failure("foobar"), NA)` 
worked. Maybe I have not written enough tests...

Reply:
I think that's fine:

```
expect_error(log_failure("foobar"), NA)
```

means that it's not falling, as you pass an expression that evals fine

```
expect_output(log_failure("foobar"), NA)
```

means it's not logging anything to the console, that's also fine ... it should only log on error

right?